aplt/scenarios.py

```diff
@@ -79,9 +137,16 @@ def reconnect_forever(reconnect_delay=30, run_once=0):
         yield timer_end("update.latency")
         yield wait(reconnect_delay)
         yield disconnect()
-        yield connect()
-        response = yield hello(uaid)
-        assert response["uaid"] == uaid
+        try:
+            yield connect()
+            response = yield hello(uaid)
+            assert response["uaid"] == uaid
+        except Exception as ex:
```

Comment:
Is there a specific exception that occurs here? can we narrow it down to something more strict than Exception

Reply:
Nope. Sadly, Client (and runner, and scenarios) raises a generic "Exception" for most of these.

I'll file a separate issue to make those custom Exceptions to prevent this sort of thing.